Float.TinCan/RemoteLRS.cs

```diff
@@ -725,7 +727,7 @@ async Task<LRSResponse> SaveDocument(string resource, Dictionary<string, string>
                 document.contentType = "application/json";
             }
 
-            var contentType = MediaTypeHeaderValue.Parse(document.contentType);
+            var contentType = MediaTypeHeaderValue.TryParse(document.contentType, out var contentMediaType) ? contentMediaType : MediaTypeHeaderValue.Parse("application/json");
```

Comment:
The exception goes into the ether.
(Pseudo stacktrace)
``` 
RemoteLRS.Task<LRSResponse> SaveDocument {
   ...
   MediaTypeHeaderValue.Parse(document.contentType)
}

RemoteLRS.Task<LRSResponse> SaveAgentProfile(AgentProfileDocument profile) {
   ...
   return await SaveAgentProfile(profile, currentProfile == null ? RequestType.put : RequestType.post);
}

LRSProxy.Task<LRSResponse> SaveAgentProfile(AgentProfileDocument profile) {
   .....
   return await targetLRS.SaveAgentProfile(profile).ConfigureAwait(false) 
} // Does not report any where.
````

Reply:
The exception does not bubble up to anything. We could try/catch all return statements for `LRSProxy`. 
Currently because there is not an explicit catch we return Status: 204 No Content

Reply:
That appears to be an issue in `Float.TinCan.ActivityLibrary`:
https://github.com/gowithfloat/Float.TinCan.ActivityLibrary/blob/5df622093144ab5997b9872334138e4553cfce24/Float.TinCan.ActivityLibrary/TinCanActivityRunner.cs#L280

It is not returning nor awaiting the task for saving the document; since this is done in an event subscriber, task failures will be discarded.